src/hub/tests/unit/stripe/customer/test_stripe_customer.py

```diff
@@ -9,9 +9,12 @@
 import flask
 import stripe
 import requests
+import connexion
+from flask import g
 
 from mock import patch
 from mockito import when, mock, unstub
+from hub.app import create_app
```

Comment:
group our `from something import something` code together, separate from external libraries like `mock` and `mockito`

Reply:
done